include/zephyr/bluetooth/audio/audio.h

```diff
@@ -219,8 +219,13 @@ struct bt_codec_data {
 /** @brief Location values for BT Audio.
  *
  * These values are defined by the Generic Audio Assigned Numbers, bluetooth.com
+ *
+ * As Audio Location is not mandatory, for handling lack of this field
+ * in codec configuration we introduce special opcode with unspecified location.
+ *
```

Comment:
I guess this comment describes `BT_AUDIO_LOCATION_UNSPECIFIED`, not `enum bt_audio_location`, so you could put it right before `BT_AUDIO_LOCATION_UNSPECIFIED`

Reply:
Done